packages/Responsive/Responsive.md

```diff
@@ -65,9 +67,63 @@ initialState = {
 </div>
 ```
 
-### Testing responsive behaviour
+#### Testing responsive behaviour
 
 Moving the responsive behaviour into JavaScript enables testing of the results of media queries, which is impossible with CSS-based media queries. It is straightforwards to mock or stub the result of a media query for testing environments. Here are some examples of how we incorporated unit tests for responsive behaviour in some of the TDS components:
 
 * [Button component tests](https://github.com/telusdigital/tds/blob/b2108d1074383ba887c5b87a2c3055799937fcd3/packages/Button/__tests__/Button.spec.jsx#L52-L68) and its corresponding [JSX](https://github.com/telusdigital/tds/blob/b2108d1074383ba887c5b87a2c3055799937fcd3/shared/components/BaseButton/BaseButton.jsx#L17-L34) using the matches boolean flag
 * [Tooltip component tests](https://github.com/telusdigital/tds/blob/b2108d1074383ba887c5b87a2c3055799937fcd3/packages/Tooltip/__tests__/Tooltip.spec.jsx#L56-L102) and its corresponding [JSX](https://github.com/telusdigital/tds/blob/b2108d1074383ba887c5b87a2c3055799937fcd3/packages/Tooltip/Tooltip.jsx#L81-L108) rendering a particular bubble only when the media query matches
+
+#### SASS Media Queries
+
+SASS-based media queries are available, and are recommended for static sites. This style of responsiveness provides a few benefits like:
+* Prevents sudden scaling when server-side rendered components first appear
+* Prevents the need to wrap Responsive around your component
+  * This in turn removes the requirement for separate CSS classes for different screen sizes that your application must programmatically switch between.
+
+However, using this implementation means losing the ability to programmatically change what is rendered through JSX on resize, so keep the design of your component in mind.
```

Comment:
I think we don't need to go into detail about why you should use the Sass stuff. I propose that our docs treat the Sass stuff as the default and just gets right into the API, as Sass media queries are well understood.

Instead, lets make it explicit that the Responsive *component* is *not* for general purpose use and it's primary use case is if you want to ensure that a component/element does not appear in the DOM for certain screen sizes. Also mention that the Responsive component may produce DOM mismatches when used with server side rendering. Point people to the workarounds/solutions using defaultMatches and mobile-detect. But reiterate that using the Sass interface is the preferred usage.

The Github Issues about Responsive have a lot of context and background to help provide colour to this explanation.